pkgs/tools/system/systemd-journal2gelf/default.nix

```
@@ -1,19 +1,17 @@
-{ stdenv, buildGoPackage, fetchFromGitHub }:
+{ stdenv, buildGoModule, fetchFromGitHub }:
 
-buildGoPackage rec {
+buildGoModule rec {
   name = "SystemdJournal2Gelf-${version}";
```

```suggestion
  pname = "SystemdJournal2Gelf-unstable";
```

because the version is not a release, but a random commit. Also `pname`!

---

Fixed, thanks!